text/notation.tex

```diff
@@ -85,17 +84,17 @@ \subsection{Dictionaries}\label{sec:dictionaries}
   \dict{K}{V} &\equiv \big \{ \{ (k \mapsto v) \mid k \in K \wedge v \in V \} \big \}
 \end{align}
 
-To denote the active domain (\ie set of keys) of a dictionary $\mathbf{d} \in \dict{K}{V}$, we use $\keys{\mathbf{d}} \subset K$ and for the range (\ie set of values), $\mathcal{V}(\mathbf{d}) \subset V$. Formally:
+To denote the active domain (\ie set of keys) of a dictionary $\mathbf{d} \in \dict{K}{V}$, we use $\keys{\mathbf{d}} \subseteq K$ and for the range (\ie set of values), $\mathcal{V}(\mathbf{d}) \subseteq V$. Formally:
 \begin{align}
   \keys{\mathbf{d} \in \mathbb{D}} &\equiv \{\ k \mid \exists v : (k \mapsto v) \in \mathbf{d}\ \} \\
   \mathcal{V}(\mathbf{d} \in \mathbb{D}) &\equiv \{\ v \mid \exists k : (k \mapsto v) \in \mathbf{d}\ \}
 \end{align}
 
-Note that since the domain of $\mathcal{V}$ is a set, should different keys with equal values appear in the dictionary, the set will only contain one such value.
+Note that since the output of $\mathcal{V}$ is a set, should different keys with equal values appear in the dictionary, the set will only contain one such value.
```

Comment:
I'm not 100% sure of what was the intended semantic here. But if as input we have a dictionary like: 

d = {(k1 → v1), (k2 → v2), (k3 → v2) }

Then V(d) = { v1, v2 }

That is, is the output that squash equal values because it is a set